tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -38,6 +39,28 @@ def select_osd_node_name():
     return osd_node_name
 
 
+@retry(AssertionError, tries=2, delay=60)
+def check_verification_steps(old_node_name, new_node_name, old_osd_id):
```

Comment:
the function name is a bit generic and doesn't reflect that for what the validation steps are

Reply:
Fixed